.rubocop.yml

```diff
@@ -76,6 +76,10 @@ Lint/EndAlignment:
 Rails:
   Enabled: true
 
+# Checks dynamic find_by_* methods. Use find_by instead of dynamic method.
```

Comment:
What about mentioning why the cop is disabled instead of what the cop does? What the cop does is in the documentation, I would also remove it from other cops. But why something is disabled is important :wink:

Reply:
@Ana06, I added why we disabled this cop, for the others, I prefer to do it in another PR 😸